### PR TITLE
feat(examples): add conslidation examples for trx

### PR DESCRIPTION
**Pull request by derickson-bitgo**
Ticket: BG-62371

### CODE REVIEW

File: examples/js/trx/create-address.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const coin = 'ttrx';
+
+// TODO: set your wallet id
+const walletId = '';
+
+// TODO: set your access token here
+// You can get this from User Settings > Developer Options > Add Access Token
+const accessToken = '';
+
+Promise.coroutine(function *() {
```
Comment (sachushaji) [inline examples/js/trx/create-address.js:18]: Using Coroutines / blue bird is not required, we can use simple async / awaits.

File: examples/js/trx/consolidate-account-balances.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+const walletPassphrase = '';
+
+// TODO: set OTP code
+const otp = '000000';
+
+// TODO: set your access token here
+// You can get this from User Settings > Developer Options > Add Access Token
+const accessToken = '';
+
+Promise.coroutine(function *() {
```
Comment (sachushaji) [inline examples/js/trx/consolidate-account-balances.js:29]: Using Coroutines / blue bird is not required, we can use simple async / awaits.

File: examples/js/trx/create-address.js (hunk 1 above)
Comment (derickson-bitgo) [reply examples/js/trx/create-address.js]: ok, updated.

File: examples/js/trx/consolidate-account-balances.js (hunk 2 above)
Comment (derickson-bitgo) [reply examples/js/trx/consolidate-account-balances.js]: Ok, updated.

**Review by sachushaji — APPROVED** [review]
Thanks for addressing the changes

**Pull request merged**